scripts/benchmark-all.sh

```diff
@@ -9,3 +9,6 @@ bash scripts/benchmark-ganache.sh
 
 echo -e "\n--- BENCHMARKING HARDHAT ---"
 bash scripts/benchmark-hardhat.sh
+
+echo -e "\n--- BENCHMARKING BLOCKNATIVE ---"
```

Comment:
I just merged a PR that added tenderly, so it seems there's now simple conflicts to resolve here and in the makefile

Reply:
Added!